cmd/bosun/bosun.example.toml

```diff
@@ -0,0 +1,85 @@
+# Hostname will be used when links are created in templates (i.e. acknowledge links)
+Hostname = "bosun.example.com"
+
+# The HTTP IP and Port to Listen on. Default is ":8070"
+HTTPListen = ":8080"
+
+# Alert checks are run by default every CheckFrequency * DefaultRunEvery. RunEvery can be overridden
+# by indivdual alerts. Defaults are "5m" and 1
+CheckFrequency = "1m"
+DefaultRunEvery = 5
+
+# Path to the rule file (file that contains definitions for alerts, macros, lookups, templates, and notifications)
+RuleFilePath = "/Users/kbrandt/src/testProdRepo/prod.conf"
+
+# timeanddate.com zones (only for use in the UI)
+TimeAndDate = [ 202, 75, 179, 136 ]
+
+# An API key for generating goo.gl shortlinks
+ShortURLKey = "aKey"
+
+# The minumum amount of alerts to create an alert on the dashboard. Default is 5
+MinGroupSize = 5
+
+# How many unknown alerts in a check cycle are needed before a group notiofication is created
+UnknownThreshold = 5
+
+# This makes it so Bosun ping's and records a metric for every value of the "host" tag it has seen. Default is false
+Ping = true
+
+# How long before hosts stop being pinged if we haven't seen a tagset for that host.Alert. Default is 24 hours
+PingDuration = "24h"
+
+# How long certain items and metrics should be displayed in the UI if we haven't seen them. Default 3 days
+SearchSince = "72h"
```

Comment:
@baweaver the tests also reference this file, so some things I set to non defaults.